File: app/src/main/java/vn/com/basc/inco/model/DetailTaskComponent.java
package vn.com.basc.inco.model;

import java.util.List;

/**
 * Created by SONY on 6/13/2016.
 */
public class DetailTaskComponent extends DetailBaseComponent{
    private Detail detail;

    public Detail getDetail() {
        return detail;
    }

    public class Detail{
        String id;
        String label;
        String status;
        String priority;
        String type;
        List<User> assigendTo;

        public String getId() {
            return id;
        }

        public String getLabel() {
            return label;
        }

        public String getStatus() {
            return status;
        }

        public String getPriority() {
            return priority;
        }

        public String getType() {
            return type;
        }

        public List<User> getAssigendTo() {
            return assigendTo;
        }
    }

}

File: Inco/DetailTaskComponent.swift
//
//  DetailTaskComponent.swift
//  Inco
//
//  Created by admin on 7/27/16.
//  Copyright © 2016 Ban Sac. All rights reserved.
//

import Foundation
class DetailTaskComponent: DetailBaseComponent {
  
    override init(data:NSDictionary) {
        let extend = data.valueForKey("detail") as? NSDictionary
        self.detail = DetailTaskComponent.Detail(data: extend!)
        super.init(data: data)
    }
    var  detail:DetailTaskComponent.Detail
    class Detail{
        var id:String?
        var label:String?
        var status:String?
        var priority:String?
        var type:String?
        var assigendTo = [User]()
        init(data:NSDictionary){
            self.id = data.valueForKey("id") as? String
            self.label = data.valueForKey("label") as? String
            self.status = data.valueForKey("status") as? String
            self.priority = data.valueForKey("priority") as? String
            self.type = data.valueForKey("type") as? String
            let users = data.valueForKey("assigendTo") as? [NSDictionary]
            for user in users! { // loop through data items
                let att = user as NSDictionary
                self.assigendTo.append(User(data: att))
            }

        }
    }
    
}